functions/adjustChatCapacity.ts

```diff
@@ -14,33 +14,40 @@ import {
 
 const TokenValidator = require('twilio-flex-token-validator').functionValidator;
 
-type EnvVars = {};
+type EnvVars = {
+  TWILIO_WORKSPACE_SID: string;
+};
 
 export type Body = {
   workerSid?: string;
-  workspaceSid?: string;
-  channelSid?: string;
-  workerLimit?: number;
   adjustment?: 'increase' | 'decrease';
 };
 
-const adjustChatCapacity = async (
+export const adjustChatCapacity = async (
   context: Context<EnvVars>,
   body: Required<Body>,
 ): Promise<{ status: number; message: string }> => {
   const client = context.getTwilioClient();
 
-  const channel = await client.taskrouter
-    .workspaces(body.workspaceSid)
+  const worker = await client.taskrouter
+    .workspaces(context.TWILIO_WORKSPACE_SID)
     .workers(body.workerSid)
-    .workerChannels(body.channelSid)
     .fetch();
 
+  if (!worker) return { status: 404, message: 'Could not find worker.' };
+
+  const { maxMessageCapacity } = JSON.parse(worker.attributes);
```

Comment:
What happens if this value is missing?

Reply:
Addressed in https://github.com/tech-matters/serverless/pull/25/commits/7eefe7365dd3bb35ac6e29f0c1839c377ae8421a